src/fields/hooks/afterChange/promise.ts

```diff
@@ -65,7 +65,7 @@ export const promise = async ({
         fields: field.fields,
         operation,
         req,
-        siblingData: siblingData[field.name] as Record<string, unknown> || {},
+        siblingData: (siblingData && siblingData[field.name] as Record<string, unknown>) || {},
```

Comment:
We prefer to use `siblingData?.[field.name]` instead of `siblingData && siblingData[field.name]`.